src/classes.py

```diff
@@ -2,20 +2,54 @@
 # constructor
 
 # YOUR CODE HERE
-        
+
+
+class LatLon:
+    def __init__(self, lat, lon):  # constructor
+        self.lat = lat
+        self.lon = lon
+
 # Make a class Waypoint that can be passed parameters `name`, `lat`, and `lon` to the
 # constructor. It should inherit from LatLon. Look up the `super` method.
 
 # YOUR CODE HERE
 
+
+class Waypoint(LatLon):
+    def __init__(self, name, lat, lon):
+        # call super constructor
+        super()
+        self.name = name
+        self.lat = lat
+        self.lon = lon
```

Comment:
You didn't have to set the attributes again because you are inheriting the attributes from the LatLon class